src/OpenTelemetry.Exporter.Prometheus.AspNetCore/PrometheusAspNetCoreOptions.cs

```diff
@@ -38,5 +38,14 @@ public int ScrapeResponseCacheDurationMilliseconds
         set => this.ExporterOptions.ScrapeResponseCacheDurationMilliseconds = value;
     }
 
+    /// <summary>
+    /// Gets or sets the allowed resource attributes filter. Default value: null (no attributes allowed).
+    /// </summary>
+    public Predicate<string> AllowedResourceAttributesFilter
```

Comment:
I understand this implementation was inspired from Java's. It appears both Java and Go have implemented this as a predicate. That said, the spec is not opinionated in how this is implemented:

> The configuration SHOULD allow the user to select which resource attributes to copy (e.g. include / exclude or regular expression based).

I think I'd prefer a simple include list. Something like:

```csharp
/// <summary>
/// Gets or sets the resource attributes to include on each metric point. By default no resource attributes are included.
/// </summary>
public string[] IncludedResourceAttributes
```